src/Components/index.jsx

```diff
@@ -0,0 +1,70 @@
+import TextField from "@material-ui/core/TextField";
+import Button from "@material-ui/core/Button";
+
+import { makeStyles } from "@material-ui/core/styles";
+import { useState } from "react";
+
+const useStyles = makeStyles({
+  mainContainerList: {
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    justifyContent: "center",
+    margin: "20px",
+  },
+  buttonCreate: {
+    marginLeft: "8px",
+  },
+  card: { margin: "20px" },
+});
+
+function ToDoList() {
+  const classes = useStyles();
+
+  const [value, setValue] = useState();
+  const [items, setItems] = useState([]);
+
+  function handleOnChange(event) {
+    setValue(event.target.value);
+  }
+
+  function handleOnClick() {
+    setItems([...items, value]);
+  }
+
+  function handleOnDelete(item) {
+    const newItems = items.filter((x, i) => x !== item);
+    setItems(newItems);
+  }
+
+  return (
+    <div className={classes.mainContainerList}>
+      <TextField
+        id="outlined-basic"
+        label="Enter task"
+        variant="outlined"
+        onChange={handleOnChange}
+        value={value}
+      />
+      <Button
+        className={classes.buttonCreate}
+        variant="contained"
+        color="primary"
+        type="submit"
+        onClick={handleOnClick}
+      >
+        Create
+      </Button>
+
+      <div>
+        {items.map((item) => (
+          <DoNewItemList onClick={() => handleOnDelete(item)}>
```

Comment:
```suggestion
          <DoNewItemList onClick={() => handleOnDelete(item)}>
```

Where is that `DoNewItemList` component imported from? Can we simplify its name to `ToDo`?

Reply:
fix it